### ISSUE TITLE
MobileNetV3 and RandomHorizontalFlip() possibly causing huge dips in accuracy.

### ISSUE THREAD

**Issue by ConfusedPelican80** [dusty-nv/jetson-inference#1835]
Hello. First time opening an issue on GitHub so I apologize if this is lacking something. I am currently using:

Jetson Nano Dev Kit 4 Gb.
Python 3.6.9 
Pytorch (torch) 1.10.0
TorchVision 0.11
JetPack SDK 4.6.1-b110
Linux4Tegra (L4T)
Ubuntu 18.04.5
Docker 20.10.21
r32.7.4 Docker Container

I have been following the transfer learning tutorial and have trained a few models such as EfficientNet-B2, ResNet-18, etc with accuracy ranging from 82-88% on a plant disease classification dataset for my capstone project. However, when I tried to use the MobileNetV3_small model. I noticed that there was a constant decline in accuracy.

 After tinkering with a few options I noticed that when I leave the RandomHorizontalFlip() function in train.py uncommented and save and close the train.py file from gedit (very important step for reproducibility some reason. Not Gedit but completely saving and completely closing the train.py from text editor). The accuracy constantly dips and never seems to recover but if I comment it, it hovers around 67% before starting to increase in the 1st epoch. (not shown in image)

![Capture](https://github.com/dusty-nv/jetson-inference/assets/92357399/e1067648-db5f-49f6-890e-6a82301a7cda)
